# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../normalmap/makefile.patch
# Copyright (C) 2011 The OpenSDE Project
# Copyright (C) 2004 - 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: OpenSDE specific adaption

--- gimp-normalmap-1.2.2/Makefile.linux.orig	2011-03-22 22:39:43.912058197 +0100
+++ gimp-normalmap-1.2.2/Makefile.linux	2011-03-22 22:44:41.352072237 +0100
@@ -10,9 +10,11 @@
 
 OBJS=normalmap.o preview3d.o scale.o
 
-LIBS=`pkg-config --libs gtk+-2.0 gtkglext-1.0 gimp-2.0 gimpui-2.0` \
--L/usr/X11R6/lib -lGLEW
+LIBS=`pkg-config --libs gtk+-2.0 gtkglext-1.0 gimp-2.0 gimpui-2.0 pangox` \
+-L/usr/X11/lib -lGLEW
 
+PPATH=`$(GIMPTOOL) --gimpplugindir`
+
 all: $(TARGET)
 
 $(TARGET): $(OBJS)
@@ -23,6 +25,7 @@
 	
 install: all
 	$(GIMPTOOL) --install-bin $(TARGET)
+	install $(TARGET) $(PPATH)/plug-ins/
 		
 .c.o:
 	$(CC) -c $(CFLAGS) $<
